### PR TITLE
[ci-visibility] Add instructions for enabling Intelligent Test Runner in javascript

### PR DESCRIPTION
**Pull request by juan-fernandez**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/setup_tests/javascript.md [hunk 1]
```diff
@@ -412,6 +412,46 @@ forEach([
 When you use this approach, both the testing framework and CI Visibility can tell your tests apart.
 
 
+## Intelligent Test Runner
+
+<div class="alert alert-warning">Intelligent Test Runner is currently in closed beta.</div>
+
+Read more about Intelligent Test Runner here.
```
Comment (juan-fernandez) [inline content/en/continuous_integration/setup_tests/javascript.md:419]: I think in here we should link to a section exclusive to intelligent test runner

File: content/en/continuous_integration/setup_tests/javascript.md [hunk 2]
```diff
@@ -412,6 +412,46 @@ forEach([
...
+`DD_CIVISIBILITY_GIT_UPLOAD_ENABLED=true` (Required)
+: Flag to enable git metadata upload.<br/>
+**Default**: `false`
+
+`DD_CIVISIBILITY_ITR_ENABLED=true` (Required)
+: Flag to enable test skipping. <br/>
+**Default**: `false`
+
+### UI activation
+In addition to the environment variables above, the Intelligent Test Runner needs to be activated in [Test Service Settings][13].
```
Comment (juan-fernandez) [inline content/en/continuous_integration/setup_tests/javascript.md:445]: should we add a screenshot here? or is that material for the intelligent test runner section?

File: content/en/continuous_integration/setup_tests/javascript.md [hunk 3]
```diff
@@ -412,6 +412,52 @@ forEach([
...
+Read more about Intelligent Test Runner here.
+
+To enable Intelligent Test Runner the following environment variables need to be set:
+
+`DD_CIVISIBILITY_AGENTLESS_ENABLED=true` (Required)
+: Enables or disables Agentless mode.<br/>
+**Default**: `false`
+
+`DD_API_KEY` (Required)
+: The [Datadog API key][1] used to upload the test results.<br/>
```
Comment (cg14823) [inline content/en/continuous_integration/setup_tests/javascript.md:428]: I think the links here may be wrong number 1 is 

`[1]: https://github.com/facebook/jest/tree/main/packages/jest-circus`

File: content/en/continuous_integration/setup_tests/javascript.md [hunk 4]
```diff
@@ -412,6 +412,52 @@ forEach([
...
+**Default**: `(empty)`
+
+`DD_SITE` (Required)
+: The [Datadog site][2] to upload results to.<br/>
```
Comment (cg14823) [inline content/en/continuous_integration/setup_tests/javascript.md:432]: Same here 

```
[2]: https://github.com/facebook/jest/tree/main/packages/jest-jasmine2
```

**Review by cg14823 — APPROVED** [review]
LGTM

**Comment by jtappa** [comment]
I have copied this addition over to #15531 and will close this PR so everything's in one place.

**Pull request closed without merging**